app/models/cherrypick_task.rb

```diff
@@ -114,23 +118,23 @@ def add_control_request(batch, control_asset)
     end
 
     # Adds any consecutive list of control requests into the current_destination_plate
-    def add_any_consecutive_control_requests(control_positions, batch, control_assets)
+    def add_any_consecutive_control_requests(control_posns, batch, control_assets)
```

Comment:
Thanks for this renaming. It was very confusing to have variables and methods using same word ```control_positions```.